guides/v2.1/javascript-dev-guide/widgets/widget_modal.md

```diff
@@ -176,6 +176,32 @@ The modal widget is subscribed to the following events:
 <li><a href="#modal_opened">opened</a></li>
 </ul>
 
+These can be listened to a couple of different ways, the primary being listening to it with jQuery's [`on`](http://api.jquery.com/on/) function, or creating a property with the name of the event that you want to listen to, like so:
+
+<pre>
+$('#modal_content').modal({
+    &lt;option1&gt;: &lt;value1&gt;,
+    &lt;option2&gt;: &lt;value2&gt;,
+    ...
+});
+$( "#modal_content" ).on( "modalclosed", function() {
+  foo(bar);
+});
+</pre>
+
+Which is functionally equivalent to the following:
+
+<pre>
+$('#modal_content').modal({
+    &lt;option1&gt;: &lt;value1&gt;,
+    &lt;option2&gt;: &lt;value2&gt;,
+    ...
+    closed: function(){
+       foo(bar);
+    }
+});
+</pre>
+
```

Comment:
Revise as follows:
You can listen to these events in two ways:
1. Use jQuery's [`on`](http://api.jquery.com/on/) function:
```
var modal = $('#some-element').modal({...});
    modal.on('modalclosed', function() {
    // Do some action when modal closed
});
```
2. Assign a callback as a property when creating a modal instance:
```
$('#modal_content').modal({
    ...
    closed: function (event, modalInstance) {
        // Do some action when modal closed
    }
});
```

Reply:
I've updated with the requested changes, but what i'm curious about is, you're able to directly edit my pull request, and i'm not complaining, but why have me edit in and replace what i wrote with what you just wrote when you're able to just edit it to what you want it to be anyways?